UI/src/view/point_asset/servicePointForm.js

```diff
@@ -118,7 +121,8 @@
         services +
         renderNewServiceElement() +
         '      </ul>' +
-        '    </div>';
+        '    </div>' +
+          components;
     };
 
     this.boxEvents = function(rootElement, selectedAsset, localizedTexts, authorizationPolicy, roadCollection, collection) {
```

Comment:
Unused definition warning on IntelliJ for some function input values (roadCollection, collection).

Reply:
unused parameter removed from the function